scrap.js

```diff
@@ -17,20 +19,82 @@ async function fetchLeaders(id) {
   return leaders
 }
 
-async function fetchOrgsWithLeaders() {
+async function searchGitHubOrgs(query) {
+  const token = process.env.GITHUB_TOKEN
+  const res = await fetch(
+    `https://api.github.com/search/users?q=${query}%20type:org`,
+    {
+      headers: token ? { Authorization: `token ${token}` } : {}
```

Comment:
I dont mind if a fork's build without a token envvar results in some orgs missing the link.
But the site should work without env vars.
i.e. degrading gracefully.

Orgs have the option of improving their GCI profile data to make their GitHub org easily findable, if they want to.
And forkers can add the env var to travis.

We will also be able to have a few orgs resolved using GitLab, when that enhancement is done, and then infer the GitHub from the GitLab.